src/lib.rs

```diff
@@ -141,7 +141,11 @@ pub mod dev {
     pub use crate::types::readlines::Readlines;
 
     pub use actix_http::body::{Body, BodySize, MessageBody, ResponseBody, SizedStream};
-    #[cfg(feature = "compress")]
+    #[cfg(any(
```

Comment:
new line.

Reply:
Miss that one ... Thanks for double checking.